policy/README.md

```diff
@@ -0,0 +1,33 @@
+# CNCF Security SIG Policy Team Introduction
```

Comment:
SIG Security instead of Security SIG

Reply:
Sentence case for titles: https://github.com/cncf/sig-security/blob/master/CONTRIBUTING.md#writing-style

Reply:
i think the title is caps for each word in the sentence ? nit the sig security one

Reply:
In English, the capitalization of headings is a stylistic choice.  (Book titles capitalize every major word.)  A while back we standardized on "sentence case" which is a somewhat arbitrary choice based on a style guide I used in prior open source projects.  Also, official abbreviation of the SIG name is at top of root README: SIG-Security.  So I think the consistent heading capitalization would be:

# CNCF SIG-Security policy team introduction

(looking at root README and rest of this doc, "policy team" seem to be all lower case unless (except when at start of a sentence). I think we could also drop "introduction" and shorten to:

# CNCF SIG-Security policy team

README kinda implies introduction, and while it was written as an intro, the policy team is long established, so it's more of an overview of the long-established group :)